client/packages/common/src/intl/locales/en/distribution.json

```diff
@@ -88,7 +88,7 @@
   "messages.click-to-return-to-requisitions": "Unable to find a requisition with that ID. Click OK to return to the requisition list",
   "messages.click-to-return-to-shipments": "Unable to find a shipment with that ID. Click OK to return to the shipment list",
   "messages.confirm-add-from-master-list": "Do you want to add all of the items from this master list?",
-  "messages.confirm-status-as": "Confirm status as {{status}}?",
+  "messages.confirm-status-as": "Confirm status as $t({{status}})?",
```

Comment:
oops! I had 'corrected' these in another PR and I now see that they are supposed to be like this 🙄